jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/basic/BasicFiltersViewImpl.java

```diff
@@ -111,6 +112,9 @@
     @Inject
     private ManagedInstance<DateRangePicker> dateRangePickerProvider;
 
+    @Inject
+    SanitizedNumberInput sanitizedNumberInput;
```

Comment:
You have to use _ManagedInstance_ in here because this is used to create multiple inputs.